assets/js/monaco/index.tsx

```diff
@@ -1,8 +1,10 @@
 import React, { useRef, useCallback } from 'react';
 import ResizeObserver from 'rc-resize-observer';
-import Editor, { Monaco } from '@monaco-editor/react';
+import Editor, { Monaco, loader } from '@monaco-editor/react';
 import type { editor } from 'monaco-editor';
 
+loader.config({ paths: { vs: '/assets/monaco-editor/vs' } });
```

Comment:
What I'm doing here is just specifying where to get the monaco files. By default, it points to the `cdn`: https://github.com/suren-atoyan/monaco-loader/blob/master/src/config/index.js .

@josephjclark this is what I wanted your feedback on.

Reply:
This is perfect :100: